diff --git a/tools/testing/selftests/kvm/Makefile.kvm b/tools/testing/selftests/kvm/Makefile.kvm
index 38c1dd752075..c3c1b437089e 100644
--- a/tools/testing/selftests/kvm/Makefile.kvm
+++ b/tools/testing/selftests/kvm/Makefile.kvm
@@ -66,6 +66,7 @@ TEST_GEN_PROGS_COMMON += kvm_page_table_test
 TEST_GEN_PROGS_COMMON += set_memory_region_test
 TEST_GEN_PROGS_COMMON += srcu_kvm_repro_test
 TEST_GEN_PROGS_COMMON += srcu_kvm_repro_delay_split_test
+TEST_GEN_PROGS_COMMON += srcu_fixed_profile_storm_test
+TEST_GEN_PROGS_COMMON += srcu_fixed_profile_preempt_test
 
 # Compiled test targets
